python/sglang/srt/models/llama2.py

```diff
@@ -346,11 +346,11 @@ def load_weights(
     ):
         stacked_params_mapping = [
             # (param_name, shard_name, shard_id)
-            ("qkv_proj", "q_proj", "q"),
-            ("qkv_proj", "k_proj", "k"),
-            ("qkv_proj", "v_proj", "v"),
-            ("gate_up_proj", "gate_proj", 0),
-            ("gate_up_proj", "up_proj", 1),
+            (".qkv_proj", ".q_proj", "q"),
```

Comment:
What is the reason for this change?

Reply:
So that for example up_proj does not match gate_up_proj and turn it into gate_gate_up_proj. Makes sure only full weight names are matched. Lifted directly from vllm code.

Reply:
Here https://github.com/vllm-project/vllm/commit/96e90fdeb3c4ebacfe24513556afccb918722b7c#diff-48d2ca5476d5b776f6401436fcf015c5ce4dc1a23d2b78a09e08fb85acc3697cR385

Reply:
ok